pkg/utils/node/predicates.go

```diff
@@ -43,15 +49,11 @@ func IsPastEmptyTTL(node *v1.Node) bool {
 	return time.Now().After(ttlTime)
 }
 
-// IsEmpty returns if the node has 0 non-daemonset pods
-func IsEmpty(node *v1.Node, pods []*v1.Pod) bool {
-	for _, p := range pods {
-		if pod.HasFailed(p) {
-			continue
-		}
-		if !pod.IsOwnedByDaemonSet(p) {
-			return false
+func getNodeCondition(conditions []v1.NodeCondition, match v1.NodeConditionType) v1.NodeCondition {
```

Comment:
nit: s/match/type would read more cleanly.

Reply:
type is a keyword, so we can't use it.